src/subcommand/wallet/inscribe.rs

```diff
@@ -45,6 +50,14 @@ pub(crate) struct Inscribe {
 
 impl Inscribe {
   pub(crate) fn run(self, options: Options) -> Result {
+    if let Some(commit_fee_rate) = self.commit_fee_rate {
```

Comment:
I think this check should be removed. There are actually reasons my one might want to use a low fee rate for the commit transaction. You could use a low fee on commit and a high fee on reveal, to encourage miners to include both or neither, using CPFP.

Reply:
Removed